pkg/timeline/adaptor/controller/timeline.ts

```diff
@@ -93,6 +99,27 @@ export class TimelineController {
       accounts.map((v) => [v.getID(), v]),
     );
 
+    // [header,avatar]
+    const headerAvatarImages = new Map<AccountID, [string, string]>();
+    const followCounts = new Map<
+      AccountID,
+      { following: number; followers: number }
+    >();
+    for (const [id, _] of accountsMap) {
```

Comment:
```suggestion
    for (const id of accountsMap.keys()) {
```
余分な要素アクセスが発生しそうなので `keys()` を使いたいです.